doc/user/INTERACTIONS.md

```diff
@@ -67,7 +67,7 @@ Other hotkeys and interactions are available:
 
 * <kbd>H</kbd>: key to toggle the display of a cheat sheet showing all these hotkeys and their statuses.
 * <kbd>?</kbd>: key to print scene description to the terminal.
-* <kbd>Esc</kbd>: close the window and quit F3D.
+* <kbd>Esc</kbd>: display/hide the console.
```

Comment:
Should we have a CLI option to open the console ?

Reply:
Hmmm I don't see why a user would want to have the console opened at start tho.

Reply:
Yeah, same as the cheatsheet.